src/SplitFlapDisplay/styles.module.scss

```diff
@@ -1,12 +1,18 @@
 .wrapper {
   --border-color: #ddd;
+  --border-radius: 10px;  // Variable for border radius
   --border-width: 1px;
   --color: #ddd;
+  --font-family: Arial, sans-serif;  // Variable for font family
   --font-size: 1rem;
+  --padding: 10px;
```

Comment:
I think this type of overall padding is best left to the parent container of this component, and not imposed by the library. please 🔪